api/v1alpha1/githubissue_types.go

```diff
@@ -40,6 +40,10 @@ type GithubIssueSpec struct {
 // GithubIssueStatus defines the observed state of GithubIssue
 type GithubIssueStatus struct {
 	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
+
+	// Number is the linked ticket number
+	// +kubebuilder:default:=0
+	Number int64 `json:"number"`
```

Comment:
We can probably do a bit better than just `Number`

Reply:
I agree